test/generate_homebrew_formula_test.sh

```diff
@@ -0,0 +1,241 @@
+#!/bin/bash
```

Comment:
I ended up using a bash script to test the bash scripts, but I am open to other options.